### DISCUSSION TITLE
How to install the secureCodeBox completely offline

### DISCUSSION THREAD

**Discussion by cnsino** [secureCodeBox/secureCodeBox#1683] [Q&A]
How to install the tool completely offline and what downloads are required

**Comment by J12934** [comment]
Hi

you should be able to install it without internet access.

1. You'll need to down load the helm chart for the operator and the scanners & hooks you want to use.
2. Replicate the image for operator and lurker (for the used version) into your local registry. (docker.io/securecodebox/operator & docker.io/securecodebox/lurker)
3. Replicate the images for the the scanners you need to your local registry. (i.e. for nmap docker.io/securecodebox/scanner-nmap & docker.io/securecodebox/parser-nmap) you can find the images used in the helm values, i.e. here: https://github.com/secureCodeBox/secureCodeBox/blob/main/scanners/nmap/values.yaml
4. If you are planning on using hooks same things for the hook images.

**Reply by cnsino** [reply]
Already installed on the machine “docker.io/securecodebox/operator & docker.io/securecodebox/lurker” .Then what to do (or what command to execute) to complete the result after the command operation similar to "helm --namespace securecodebox-system upgrade --install securecodebox-operator secureCodeBox/operator"
